lib/dockerize_rails/docker_commands/docker_start.rb

```diff
@@ -0,0 +1,77 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStart
+      def self.start_rails
+        services = DockerizeRails::DockerCommands::Helpers.get_services_from_docker_compose
+        docker_start(services['rails'], "#{DRConfig.application_name}_rails:#{DRConfig.application_env}") \
+        if services.has_key? 'rails'
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.start_mysql
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          services = DockerizeRails::DockerCommands::Helpers.get_services_from_docker_compose
+          docker_start(services['mysql'], "#{DRConfig.application_name}_mysql:#{DRConfig.application_env}") \
+          if services.has_key? 'mysql'
+        end
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.start_postgres
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          services = DockerizeRails::DockerCommands::Helpers.get_services_from_docker_compose
+          docker_start(services['postgresql'], "#{DRConfig.application_name}_postgres:#{DRConfig.application_env}") \
+          if services.has_key? 'postgresql'
+        end
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.docker_start(service, image_name)
+        options = {}
+        options['Image'] = image_name
+        options['name'] = "#{image_name.split(':')[0]}_container"
+        options['ExposedPorts'] = { "#{service['expose']}/tcp" => {} } if service.has_key? 'expose'
+        if service.has_key? 'ports'
```

Comment:
__[rubocop]__
* Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->